modules/nw-ingress-integrating-route-secret-certificate.adoc

```diff
@@ -0,0 +1,79 @@
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-integrating-route-secret-certificate_{context}"]
+= Securing route with external certificates in TLS secrets
+
+:FeatureName: Securing route with external certificates in TLS secrets
+include::snippets/technology-preview.adoc[]
+
+The following workflow describes how to integrate third-party certificate management solutions, using the certificate reference field outlined under API Extensions.
```

Comment:
>API Extensions

Please cross-check the case standards here.

Reply:
Noted. I will change accordingly.